commons/src/main/java/cloud/orbit/concurrent/Task.java

```diff
@@ -93,6 +93,20 @@ public Task()
         }
     }
 
+    public static void setCommonPool(final Executor commonPool)
+    {
+        Task.commonPool = commonPool;
+    }
+
+    public static void setSchedulerExecutor(final ScheduledExecutorService schedulerExecutor)
+    {
+        Task.schedulerExecutor = schedulerExecutor;
+    }
+
+    public static void shutdownSchedulerExecutor() {
+        schedulerExecutor.shutdown();
```

Comment:
Not 100% sold on this but it prevents the JVM from exiting and if I set a custom thread factory (setting threads to daemon) cloud.orbit.concurrent.test.TaskImplementationTest#testSleep fails to run..